src/main/java/duke/Task.java

```diff
@@ -0,0 +1,37 @@
+package duke;
+
+/**
+ * Represents task object
```

Comment:
I noticed that there are some places where the comments do not end with a full stop. (e.g in this Task.java file and TaskList.java) Would it be better to add a full stop at the end of every comment?